## Hi there ðŸ‘‹

ðŸ”¬ Bioinformatician | Data Scientist | Computational Biologist | Clinical Data Analyst | Biostatististician

I am a bioinformatician with 10+ years of experience in genomics, transcriptomics, and computational biology, focused on leveraging big data to solve complex biological questions. With a strong foundation in genomic data analysis, statistical modeling, and machine learning, I specialize in Next-Generation Sequencing (NGS), single-cell RNA-seq, GeomaX, Spatial Transcriptomic and multi-omics data integration.

## Featured Project: Clinical Kinase Index (CKI) Web Application

Developed an interactive R Shiny web application supporting precision medicine research focused on the Clinical Kinase Index (CKI). The tool integrates multi-omics and clinical datasets to enable researchers and clinicians to dynamically explore kinase gene expression, clinical associations, and survival outcomes across various cancers.

### Key features include:

	Interactive mRNA expression heatmaps of all kinases, including understudied ones
	Filterable tables with kinase gene names, cancer types, CKI scores, rankings, and metadata
	Gene-specific exploration pages with filters by Target Development Level (TDL), kinase type/group, and mechanism of action (MOA) targets
	Cancer-specific disease pages showing volcano plots of differentially expressed kinases
	Clinical data visualizations such as boxplots for expression by TNM staging and Kaplan-Meier survival curves
	Data download options for reproduction and further analysis, including mutation and copy number alteration (CNA) data
	Built entirely in R (version 3.3+) using packages like shiny, TCGAbiolinks, limma, edgeR, survival, plotly, and many more. The app leverages Bioconductor databases 	and is designed for scalable, reproducible bioinformatics workflows to facilitate translational cancer research.

Codebase and additional resources are openly available at github : GitHub - Schurer Lab CKI
Live app accessible at cki.ccs.miami.edu and shinyapps.io/CKIApp.


ðŸ”§ Technical Skills:

	Programming: Python, R, Bash, SQL
	Data Analysis: RNA-seq, ChIP-seq, ATAC-seq, WGS, GWAS
	Tools & Libraries: DESeq2, edgeR, STAR, HISAT2, GATK, Seurat, PyTorch, TensorFlow
	Tech Stack: Docker, Nextflow, SLURM, Git
	Platforms: AWS, GCP, Linux

 
ðŸ§  Research Interests:

	Single-cell genomics
	Multi-omics integration
	AI/ML in computational biology
	Biomarker discovery
	Personalized medicine
 	Spatial Transcriptomics



 
ðŸŒ± I am passionate about open-source bioinformatics tools, reproducible research, and building pipelines that make complex bioinformatics analysis accessible to both researchers and clinicians. Currently, I am exploring deep learning approaches for predictive modeling in genomic data.


ðŸ”— Feel free to browse my repositories to check out the bioinformatics pipelines, scripts, and tools Iâ€™ve developed. Let's collaborate on pushing the boundaries of bioinformatics and computational biology!
